data/FMI/fmi1.osm

```diff
@@ -1486,9 +1486,7 @@
     <tag k='one:attachment_point' v='1' />
   </node>
   <node id='-147464' action='modify' visible='true' lat='48.26257759224' lon='11.66825824905' />
-  <node id='-147466' action='modify' visible='true' lat='48.26254899224' lon='11.66849904905'>
-    <tag k='one:portal' v='elevator1' />
```

Comment:
why is the elevator gone now?

Reply:
It was dysfunctional. It wasn't connected to any other nodes.

Reply:
ah, I see ;-)